hedera-node/hedera-mono-service/src/test/java/com/hedera/test/utils/SeededPropertySource.java

```diff
@@ -114,19 +115,22 @@ public EvmFnResult nextEvmResult() {
         // call nextStateChanges(), even though state changes are not part of EvmFnResult anymore,
         // in order to advance SEEDED_RANDOM and get correct values for subsequent fields
         nextStateChanges(2, 5);
-        return new EvmFnResult(
+        final var evmResult = new EvmFnResult(
                 contractId,
                 result,
                 error,
                 bloom,
                 gasUsed,
                 logs,
                 createdContractIds,
+                List.of(),
                 evmAddress,
                 nextUnsignedLong(),
                 nextUnsignedLong(),
                 nextBytes(128),
                 nextEntityId());
+        evmResult.setContractNonces(nextContractNonces(2));
```

Comment:
I don't get why this setting of the nonces just isn't up there at L126?

Reply:
Because regression tests, and especially `deserializationWorksForAllSupportedVersions` was failing